src/core/utils/server.js

```diff
@@ -0,0 +1,63 @@
+/* @flow */
+import path from 'path';
+
+import fs from 'fs-extra';
+import config from 'config';
+import type { $Request, $Response } from 'express';
+
+import log from 'core/logger';
+
+const PREFIX = 'enableFeature';
+
+export const getFeatureFlagName = (featureFlagKey: string): string => {
+  return featureFlagKey.replace(PREFIX, '');
+};
+
+type ExpressHandler = (req: $Request, res: $Response) => void;
+
+type ViewFrontendVersionHandlerParams = {|
+  _config?: typeof config,
+  _log?: typeof log,
+  versionFilename?: string,
+|};
+
+export const viewFrontendVersionHandler = ({
+  _config = config,
+  _log = log,
+  // This is a magic file that gets written by deployment scripts.
+  versionFilename = 'version.json',
+}: ViewFrontendVersionHandlerParams = {}): ExpressHandler => {
+  const version = path.join(_config.get('basePath'), versionFilename);
+
+  const featureFlags = Object.keys(_config)
+    .filter((key) => {
+      return key.startsWith(PREFIX);
+    })
+    .reduce((map, key) => {
+      return {
+        ...map,
+        [getFeatureFlagName(key)]: _config.get(key),
```

Comment:
Can this just be `[key]: _config.get(key)`? It will still be readable. I think this would be better because someone may think to copy/paste what they see in the JSON and grep for it in the code.

Reply:
Yep, although `grep` would still work as it is only about removing a prefix, but why not.

Reply:
Thinking about it again, I am not sure it is wise to keep the (internal) prefix.